bangtt/fresher_user/controllers/Home.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+class Home extends Base_Controller {
+    public function index() {
+        $this->view("home", "index");
+    }
+
+    public function add_user() {
+        if (isset($_POST['btnCreate'])) {
+            date_default_timezone_set('Asia/Ho_Chi_Minh');
+            $username = $_POST['txtUsername'];
+            $email = $_POST['txtEmail'];
+            $password = $_POST['txtPassword'];
+            $select = $_POST['select'];
+            if ($select == '1') {
+                $status = 1;
+            } else {
+                $status = 0;
+            }
+            $dateNow = new DateTime();
+            $date_format_date_now = date_format($dateNow, 'Y-m-d H:I:S');
```

Comment:
Format date sai `Y-m-d H:I:S => Y-m-d H:i:s`
S - English ordinal suffix for the day of the month, 2 characters
I (capital i) - Whether or not the date is in daylight saving time

Có thể dùng `$date_format_date_now  = date('Y-m-d H:i:s');`

Variable `$user_created_time` và  `$user_updated_time` không cần thiết, có thể sử dụng luôn `$date_format_date_now`